ZAPD/ZTexture.h

```diff
@@ -18,6 +18,14 @@ enum class TextureType
 	GrayscaleAlpha16bpp,
 };
 
+enum class TextureTypeSize : uint8_t
+{
+	TEX_TYPE_8 = 8,
+	TEX_TYPE_16 = 16,
+	TEX_TYPE_32 = 32,
+	TEX_TYPE_64 = 64,
+};
+
```

Comment:
For the sake of consistency with the other enums, could you make these enum entries be CamelCase?

Something like:

```
enum class TextureTypeSize : uint8_t
{
	Size8 = 8,
	Size16 = 16,
	Size32 = 32,
	Size64 = 64,
};
```

I removed the `TEX_TYPE_` prefix because that information is already covered in `TextureTypeSize::`.